version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.4.22",
```

Comment:
Was this a bug that it was different ?

Reply:
Is there other "1.4.22" is source code that need to be updated:
![image](https://github.com/microsoft/security-utilities/assets/81775155/7ce98d71-0caa-4080-b5da-8c2475babbc0)

Reply:
@shaopeng-gh yes. we missed updating the version.

Reply:
The test show all passed, was the test not effective, or actually testing something else not related
![image](https://github.com/microsoft/security-utilities/assets/81775155/0991f589-5ce0-4932-b613-41850e052338)

Reply:
@shaopeng-gh This test does not seem to execute. But it's a valid concern and I will follow up on what the deal here is.